examples_stardust/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "utils"
```

Comment:
That was done so the imports in the examples read a bit nicer, e.g. `use utils::get_address_with_funds;`. But it's fine either way. I'll change it.

Reply:
Yeah, if it's linked to the `utils` imports name then might not be worth it. I do think it would be better to have explicit calls like `examples::get_address_with_funds(...)` so it's clear that it's not from the library though.

Reply:
Agreed, `examples` is better. Had to rename the old `examples` package to `legacy_examples` to avoid the name conflict, but that's fine I guess.